mtgjson5/output_generator.py

```diff
@@ -19,6 +19,7 @@
     MtgjsonSetListObject,
     MtgjsonStructuresObject,
 )
+from .compiled_classes.mtgjson_all_tcg_skus import MtgjsonAllTcgSkusObject
```

Comment:
This should be imported directly from compiled_classes, did you add this import to __init__ in the compiled_classes dir?